shell/app/modules/msp/pages/gateway/containers/api-package-create.tsx

```diff
@@ -308,7 +301,7 @@ export const PureApiPackage = () => {
           }
         }}
       >
-        <RenderPureForm form={basicForm} {...formProps} list={fieldsList} />
+        <RenderPureForm form={basicForm} {...formProps} list={fieldsList} onlyItems />
```

Comment:
will `onlyItems` cause issue?

Reply:
Because it has a Form around it, only need Items inside.